documentation/concepts.md

```diff
@@ -0,0 +1,33 @@
+# Namespaces
```

Comment:
The associated features are not yet merged into remastered branch. Denote for each line whether it is merged or planned.